src/Lykke.Service.BlockchainWallets.Core/Services/IBlockchainIntegrationService.cs

```diff
@@ -12,5 +13,7 @@ public interface IBlockchainIntegrationService
         IBlockchainApiClient TryGetApiClient(string blockchainType);
 
         IBlockchainApiClient GetApiClient(string blockchainType);
+
+        ImmutableDictionary<string, BlockchainApiClient>.Enumerator GetApiClientsEnumerator();
```

Comment:
what is the reason of usage ImmutableDictionary.Enumerator structure

Reply:
I need to go through the list of BIL service clients at initialization step (see [below](https://github.com/LykkeCity/Lykke.Service.BlockchainWallets/pull/41/files#diff-099a1a05bfc044ef251b21f53310d833R50)). I need some way to access the list, and I can imagine 2 ways to do it:
* expose the `private readonly` collection from `BlockchainIntegrationService` and break incapsulation or
* provide a public enumerator to it trying to keep incapsulation principle.

I simply choose the second.